test/helper.rb

```diff
@@ -78,19 +78,20 @@ def run
             }
           end
         end
+        if respond_to? :clean_tmp_paths
+          clean_tmp_paths
+        end
       end
     end
 
     Minitest::Result.from self # per contract
   end
 end
 
-if ENV['CI']
-  Minitest::Test.prepend TimeoutEveryTestCase
+Minitest::Test.prepend TimeoutEveryTestCase
 
-  require 'minitest/retry'
-  Minitest::Retry.use!
-end
+require 'minitest/retry'
+Minitest::Retry.use!
```

Comment:
Ehhh, I don't think we should turn on Retry locally. That would be pretty annoying if you were trying to TDD anything. I'm fine with Timeouts being on in all environments.